jupiterone/resource_integration.go

```diff
@@ -165,7 +165,12 @@ func (r *IntegrationResource) Read(ctx context.Context, req resource.ReadRequest
 		resp.Diagnostics.AddError("Failed to marshal config", err.Error())
 		return
 	}
-	data.Config = types.StringValue(string(configJSON))
+
+	// Only update config if it doesn't contain masked values
```

Comment:
This fixes an outstanding bug wherein on update operations (2nd call to backend after initial create, the J1 api returns masked values). Storing a masked value in the state would result in plan diffs for subsequent operations that would never be resolvable/succeed, so we omit the config change from the state since it's invalid. This update has coverage by the acceptance test suite.